app/resources/css-default.template

```diff
@@ -605,6 +607,16 @@ td.submit {
   border: 2px solid #e6e6e6;
 }
 
+.query.seek .field {
```

Comment:
no. that comment is about why you are making the cl. it's not explaining anything about the style. no comment is ok